Tasks/GitHubReleaseV0/task.json

```diff
@@ -15,7 +15,7 @@
     "version": {
         "Major": 0,
         "Minor": 0,
-        "Patch": 4
+        "Patch": 6
```

Comment:
This looks to be wrong versioning to me. Update Akbar about it.

Reply:
Thanks @romil07 for making this change at priority